README.md

```diff
@@ -1,15 +1,101 @@
 # getappmap/archive-appmap
 
+> To get started with AppMap in GitHub actions, you need to start by installing the [AppMap App on the official GitHub Marketplace](https://github.com/marketplace/get-appmap)
+
```

Comment:
Add a `>` here to make it one section, as in https://github.com/getappmap/install-action#getappmapinstall-appmap-

Reply:
Ah thanks!